tests/queries/0_stateless/03237_create_or_replace_view_atomically_with_atomic_engine.sh

```diff
@@ -16,10 +16,13 @@ function create_or_replace_view_thread
 }
 export -f create_or_replace_view_thread;
 
+# Old analyzer doesn't retry in case of resolving table identifier errors
```

Comment:
I'm not even sure that https://github.com/ClickHouse/ClickHouse/commit/3fc2f551d147ee2cc7572303c5b6538395d79b1a is correct, since it should resolve only using UUID, otherwise it may end up using wrong table. @tuanpach any thoughts?

I actually broke it here https://github.com/ClickHouse/ClickHouse/pull/76308, let me think on how this can be fixed